packages/password/index.html

```diff
@@ -0,0 +1,31 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Password generator default settings</title>
+</head>
+<body>
+    <h1>Password generator default settings</h1>
+    <pre><code id="rules"></code></pre>
+    <pre><code id="parameters"></code></pre>
+    <pre><code id="password"></code></pre>
+    <script type="module">
+        import {generate} from "./index.js";
+        import {Password} from './lib/apple.password.js'
+        const pws = [];
+        for (let i = 0; i < 10; i++) {
+            pws.push(generate())
+        }
+        document.querySelector("#password").textContent = pws.join('\n');
+        document.querySelector("#rules").textContent = Password.defaults.defaultPasswordRules;
+
+        const {parameters, entropy} = new Password().parse(Password.defaults.defaultPasswordRules);
+        document.querySelector("#parameters").textContent = JSON.stringify(parameters, null, 2);
+        document.querySelector("#parameters").textContent += '\n' + "Entropy: " + entropy;
```

Comment:
Maybe cache this or make it a single line?

Reply:
I split the line out, 👍🏻